manifests/kustomize/base/pipeline-application.yaml

```diff
@@ -0,0 +1,49 @@
+    marketplace.cloud.google.com/deploy-info: '{"partner_id": "google-cloud-ai-platform", "product_id": "kubeflow-pipelines", "partner_name": "Google Cloud AI Platform"}'
+spec:
+  addOwnerRef: true
+  selector:
+    matchLabels:
+      application-crd-id: kubeflow-pipelines
+  descriptor:
+    version: 0.1.34
```

Comment:
Quick check, will our on-call script auto update this field? not familiar with that part.
It's OK if we put that change to another PR.

Reply:
We should update release playbook to also change this field

Reply:
thanks. will do once this is merged.